cards.js

```diff
@@ -342,11 +342,10 @@ DECK_DEFINITONS =
             , [true,  "21", "* %move% +0", "* %attack% +0", "** %pull% 1", "* %air%"]
             , [true,  "21", "* %move% +0", "* %attack% +0", "** %pull% 1", "* %air%"]
             , [false, "29", "* %move% +0", "* %attack% -1", "** %target% 2", "** %air%%use_element%: %push% 2"]
-            , [false, "37", "* %move% +0", "* %attack% +0 <div style='display: inline-block; width: 0; margin-top: -1.5em; vertical-align: bottom'>%aoe-4-with-black%</div>", "** %air%%use_element%: +1 %attack% <div style='display: inline-block; margin-right: -3em'>%aoe-circle-with-side-black%</div>"]
+            , [false, "37", "* %move% +0", "* %attack% +0 <div style='display: inline-block; width: 0; margin-top: -1.5em; vertical-align: bottom'>%aoe-4-with-black%</div>", "** %air%%use_element%: %attack% +1 <div style='display: inline-block; margin-right: -3em'>%aoe-circle-with-side-black%</div>"]

```

Comment:
Are we sure about it? I can ask it on FAQ, but as I understand that card is: if consume Air, performs an extra attack with different AOE. But maybe it makes sense and it's just an upgraded version of the attack. Then I have to change the regex to detect that way to write it, in case we want to say "Attack 5 (small AOE), consume_air: Attack 6 (big AOE)"